routers/logout_filter.go

```diff
@@ -0,0 +1,42 @@
+package routers
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/beego/beego/context"
+	"github.com/beego/beego/logs"
+	"github.com/casdoor/casdoor/conf"
+)
+
+var (
+	logoutMinutes   = time.Minute * 30
+	cookie2LastTime sync.Map
+)
+
+func init() {
+	logoutMinutes_int, err := conf.GetConfigInt64("logoutMinutes")
```

Comment:
use logoutMinutesInt

Reply:
I suggest following the existing code, use string, which supports empty:

```go
func (org *Organization) GetInitScore() (int, error) {
	if org != nil {
		return org.InitScore, nil
	} else {
		return strconv.Atoi(conf.GetConfigString("initScore"))
	}
}
```